hasher-matcher-actioner/hmalib/lambdas/integrations.py

```diff
@@ -0,0 +1,200 @@
+import json
+import hmac
+import typing as t
+from dataclasses import dataclass
+
+from hmalib.common.logging import get_logger
+
+import requests
+
+logger = get_logger()
+
+# Placeholder values not used in prod so that we dont leak anything to github
+API_TOKEN = "3156067"
+WORKPLACE_APP_SECRET = "70856018561"
+HMA_API_GATEWAY_URL = "https://xxxxxxxxx.execute-api.us-east-1.amazonaws.com"
+
+
+@dataclass
+class PhotoToUpload:
+    photo_id: str
+    photo_url: str
+    additional_fields: t.List[str]
+
+
+def lambda_handler(event, context):
+    logger.info(event)
+
+    if not can_process_request(event):
+        logger.info("Cannot Process Request")
+        return {"statusCode": 400, "body": json.dumps(event)}
+    if not authorize_request(event):
+        logger.info("Cannot Authorize Request")
+        return {"statusCode": 400, "body": json.dumps("Cannot Authorize Request")}
+    response, content_to_upload = process_request(event)
+    for content in content_to_upload:
+        print(f"Uploading content with ID {content.photo_id} to HMA")
+        upload_to_HMA(content)
+
+    return response
+
+
+def upload_to_HMA(content: PhotoToUpload):
+    submit_payload = {
+        "content_id": content.photo_id,
+        "content_type": "photo",
+        "content_url": content.photo_url,
+        "additional_fields": content.additional_fields,
+        "force_resubmit": True,  # For testing, allow duplicate submission
+    }
+
+    payload_bytes = json.dumps(submit_payload).encode()
+
+    response = requests.post(
+        url=HMA_API_GATEWAY_URL + "/submit/url/",
+        headers={
+            "Content-Type": "application/json",
+            "Authorization": API_TOKEN,
+        },
+        data=payload_bytes,
+    )
+    response.raise_for_status()
+
+    logger.info(response.json())
+
+
+def get_bytes_from_url(url: str):
+    r = requests.get(url)
+    r.raise_for_status()
+    return r.content
+
+
+def process_request(
```

Comment:
All pretty workplace specific but the idea of having different content_ids and image_ids my be a useful distinction to highlight in an example